### DISCUSSION TITLE
Remaining REST endpoints

### DISCUSSION THREAD

**Discussion by peepo5** [fabianboesiger/ftx#58] [General]
This is a place to discuss adding the remaining REST endpoints.
The current ones I am interested in adding is the future pairs as they are very useful.
I only have basic knowledge of rust, but I expect over time I can contribute to these remaining endpoints.
:)

**Comment by fabianboesiger** [comment]
This is actually a good project to contribute if you are new to Rust.
Just take a look at similar endpoints, and change them such that they work for the endpoint you want to implement.
Feel free to submit a PR, worst thing that happens is I'll give you feedback :)
